react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -1,7 +1,10 @@
 import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
 import store from '@redux/store';
+import actionsCreators from '@redux/book/actions';
```

Comment:
On some component you can use various actions so I recommend change this name for something more descriptive for example bookActions